### PR TITLE
[DE] Added cover and a few other sentences

### PR DESCRIPTION
**Pull request by IceBeach7**

### CODE REVIEW

File: sentences/de/_common.yaml [hunk 1]
```diff
@@ -16,5 +16,6 @@ expansion_rules:
   area: "(in (der | dem) | im) {area}"
   turn_on: "(an | ein)"
   turn_off: "(ab | aus)"
+  cover_variants: "(das tor | (die | alle) tore | das rollo | (die | alle) rollos | die abdeckung | (die | alle) abdeckungen)"
```
Comment (ThomDietrich) [inline sentences/de/_common.yaml:19]: Imho nicht vollständig und Substantive sollten großgeschrieben sein

**Comment by synesthesiam** [comment]
@Scorpoon can you take a look at these changes please?

File: sentences/de/_common.yaml [hunk 2]
```diff
@@ -16,5 +16,6 @@ expansion_rules:
   area: "(in (der | dem) | im) {area}"
   turn_on: "(an | ein)"
   turn_off: "(ab | aus)"
+  cover_variants: "(das Tor | (die | alle) Tore | die Garage | (die | alle) Garagen | das Garagentor | (die | alle) Garagentore | das Rollo | (die | alle) Rollos | die Abdeckung | (die | alle) Abdeckungen)"
```
Comment (kroimon) [inline sentences/de/_common.yaml:19]: I would also add "Rollladen" (roller shutter) here as this is probably a very common cover to control.

File: sentences/de/cover_HassCloseCover.yaml [hunk 3]
```diff
@@ -2,4 +2,6 @@ language: de
 intents:
   HassCloseCover:
     data:
-      - sentences: []
+      - sentences:
+          - "schließe <area> <cover_variants>"
+          - "schließe <cover_variants> <area>"
```
Comment (kroimon) [inline sentences/de/cover_HassCloseCover.yaml:7]: If you are adding the "(mach | mache) ... <turn_on/turn_off>" variants to other domains, I suggest also adding this to the covers, e.g. "mache die Garage zu".
Even if "mache <something>" is not the highest form of German, I guess it it used quite often.

File: sentences/de/fan_HassTurnOff.yaml [hunk 4]
```diff
@@ -5,6 +5,7 @@ intents:
       - sentences:
           - "(schalte | stoppe) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_off>]"
           - "(schalte | stoppe) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
+          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
```
Comment (kroimon) [inline sentences/de/fan_HassTurnOff.yaml:8]: Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_off>]"
          - "(mach | mache) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_off>]"
```

File: sentences/de/fan_HassTurnOn.yaml [hunk 5]
```diff
@@ -5,6 +5,7 @@ intents:
       - sentences:
           - "(schalte | starte) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_on>]"
           - "(schalte | starte) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
+          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
```
Comment (kroimon) [inline sentences/de/fan_HassTurnOn.yaml:8]: Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (den Ventilator | (die | alle) Ventilatoren) [<turn_on>]"
          - "(mach | mache) (den Ventilator | (die | alle) Ventilatoren) <area> [<turn_on>]"
```

File: sentences/de/light_HassTurnOff.yaml [hunk 6]
```diff
@@ -5,5 +5,6 @@ intents:
       - sentences:
           - "Schalte (das Licht | (die | alle) Lichter) <area> <turn_off>"
           - "Schalte <area> (das Licht | (die | alle) Lichter) <turn_off>"
+          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_off>"
```
Comment (kroimon) [inline sentences/de/light_HassTurnOff.yaml:8]: Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_off>"
          - "(mach | mache) (das Licht | (die | alle) Lichter) <area> <turn_off>"
```

File: sentences/de/light_HassTurnOn.yaml [hunk 7]
```diff
@@ -5,5 +5,6 @@ intents:
       - sentences:
           - "Schalte (das Licht | (die | alle) Lichter) <area> <turn_on>"
           - "Schalte <area> (das Licht | (die | alle) Lichter) <turn_on>"
+          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_on>"
```
Comment (kroimon) [inline sentences/de/light_HassTurnOn.yaml:8]: Similar to the existing sentences above:
```suggestion
          - "(mach | mache) <area> (das Licht | (die | alle) Lichter) <turn_on>"
          - "(mach | mache) (das Licht | (die | alle) Lichter) <area> <turn_on>"
```

File: sentences/de/cover_HassOpenCover.yaml [hunk 8]
```diff
@@ -2,4 +2,6 @@ language: de
 intents:
   HassOpenCover:
     data:
-      - sentences: []
+      - sentences:
+          - "öffne <area> <cover_variants>"
+          - "öffne <area> <cover_variants>"
```
Comment (kroimon) [inline sentences/de/cover_HassOpenCover.yaml:7]: This is a duplicate and should probably be:
```suggestion
          - "öffne <area> <cover_variants>"
          - "öffne <cover_variants> <area>"
```

**Review by kroimon — CHANGES_REQUESTED** [review]

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @IceBeach7 

thanks for this contribution 👍 

You need to rebase your branch to the latest `main` so you will get the newest expansion rules. With these you can adjust the `homeassistant` and `light` sentences. The `cover` and `fan` sentences were meanwhile added by #92, so you can skip them in this PR.

regards,
Michael

**Comment by kroimon** [comment]
I think this is now obsolete as it was basically the base for #92

**Comment by mib1185** [comment]
there are still open additions to the `homeassistant` and `light` sentences, which were not covered by #92 (_or any later PR_)

**Comment by balloob** [comment]
This PR has gone stale. Please open a new PR without conflicts if you want to incorporate these changes.

**Pull request closed without merging**